app/views/dashboard.erb

```diff
@@ -6,7 +6,7 @@
         <% if dashboard.current_exercises.any? %>
             <p>Keep working on these problems, or archive them.</p>
             <ul class="nav nav-stacked" id="menu-current" role="menu">
-              <% dashboard.current_exercises.limit(5).each do |exercise| %>
+              <% dashboard.current_exercises.reverse.limit(5).each do |exercise| %>
```

Comment:
By activity is a "scope" that changes the query sort order: https://github.com/exercism/exercism.io/blob/master/lib/exercism/user_exercise.rb#L19